src/data-table/__tests__/column-boolean.test.js

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {act} from 'react-dom/test-utils.js';
+
+import {BooleanColumn} from '../index.js';
+
+let container: HTMLDivElement;
+
+describe('boolean column', () => {
+  beforeEach(() => {
+    if (__BROWSER__) {
+      container = document.createElement('div');
+      if (document.body) {
+        document.body.appendChild(container);
+      }
+    }
+  });
+
+  afterEach(() => {
+    if (__BROWSER__) {
+      if (document.body && container) {
+        document.body.removeChild(container);
+      }
+    }
+  });
+
+  it('is sortable by default', () => {
+    const column = BooleanColumn({title: 'column'});
+    expect(column.sortable).toBe(true);
+  });
+
+  it('is filterable by default', () => {
+    const column = BooleanColumn({title: 'column'});
+    expect(column.filterable).toBe(true);
+  });
+
+  it('applies provided sortable value', () => {
+    const column = BooleanColumn({title: 'column', sortable: false});
+    expect(column.sortable).toBe(false);
+  });
+
+  it('applies provided filterable value', () => {
+    const column = BooleanColumn({title: 'column', filterable: false});
+    expect(column.filterable).toBe(false);
+  });
+
+  it('cell renders T if true value provided', () => {
+    const column = BooleanColumn({title: 'column'});
+    const Cell = column.renderCell;
+
+    act(() => {
+      ReactDOM.render(<Cell value={true} />, container);
+    });
+
+    const cell = container.querySelector('div');
+    if (cell) {
```

Comment:
how is it possible to that sometimes `cell` is defined, but sometimes not? maybe I am missing something here

Reply:
it's because `querySelector` can return a dom node or `null`. trying to access `textContent` on the object causes a flow error

Reply:
oh I see - so the else clause would fail the tests?

Reply:
correct, I suppose I could have added a flowfixme comment. maybe that's clearer?

Reply:
either that or a comment would help